sunpy/time/time.py

```diff
@@ -48,6 +48,7 @@
     "%Y:%j:%H:%M:%S",          # Example 2012:124:21:08:12
     "%Y:%j:%H:%M:%S.%f",       # Example 2012:124:21:08:12.999999
     "%Y%m%d%H%M%S",            # Example 20140101000001 (JSOC / VSO)
+    "%Y.%m.%d_%H:%M:%S_TAI",   # Example 2016.05.04_21:08:12_TAI
```

Comment:
I am happy to add this here, but you should know we assume all times are UTC so this may cause unexpected accuracy issues.